automatic/dropbox/tools/functions.ps1

```diff
@@ -1,5 +1,6 @@
 ﻿Function getDropboxRegProps() {
-  $uninstallRegistryPath = 'HKCU:\Software\Microsoft\Windows\CurrentVersion\Uninstall\Dropbox'
+  $bitness = @{$true = "\WOW6432Node"; $false = ""}[ (Get-ProcessorBits) -eq 64 ]

```

Comment:
Sorry my mistake, I just checked it on a stern with windows 2003 server with this from the PowerShell command prompt
`$some=$true;$bitness = @{$true = "\WOW6432Node"; $false = "false"}[ $some ]; echo "bitness is "$bitness`
The answer was `\WOW6432Node` as you would expect.